### PR TITLE
Longer wait for resource DNS propagation

### PR DESCRIPTION
**Pull request by christothes**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/confidentialledger/test-resources-post.ps1 [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# Confidential Ledger resources can take up to 60 seconds to resolve in DNS after being newly provisioned.
-Write-Host 'Waiting 60 seconds to let the new resource propagate to DNS'
-Start-Sleep -Seconds 60
+# Confidential Ledger resources can take a few minutes to resolve in DNS after being newly provisioned.
```
Comment (pallavit) [inline sdk/confidentialledger/test-resources-post.ps1:1]: Should we consider using shared resource instead of provisioning it every time? Assuming CL resources are not that costly to keep alive.

Comment (pallavit) [reply sdk/confidentialledger/test-resources-post.ps1:1]: Cognitive services (TA, FR) use shared resources by adding the secrets directly to the pipeline and test-resources is basically empty. Not saying that is the best solution here since we are only waiting a couple of mins but it may be worth considering if this does not fix things either.

Comment (JoshLove-msft) [reply sdk/confidentialledger/test-resources-post.ps1:1]: We shouldn't use static resources unless we absolutely need to. It makes it harder for contributors and it introduces additional risks.

https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/365/Static-Test-Resources

Comment (JoshLove-msft) [reply sdk/confidentialledger/test-resources-post.ps1:1]: If you want to avoid hardcoding a wait time, you can look into overriding this method - https://github.com/Azure/azure-sdk-for-net/blob/0d44555ba2e14692ba7722864690c69e548c666d/sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs#L235

**Review by JoshLove-msft — APPROVED** [review]
